docs/user-guides/workspace-access/remote-desktops.md

```diff
@@ -53,3 +53,11 @@ or use your favorite RDP client to connect to `localhost:3399`.
 ![windows-rdp](../images/ides/windows_rdp_client.png)
 
 > Note: Default username is `Administrator` and password is `coderRDP!`.
+
+## RDP Web
+
+Our WebRDP module in the Coder Registry adds a one-click button to open an RDP
+session in the browser. This requires just a few lines of Terraform in your
+template, see the documentation on our registry for setup.
```

Comment:
@stirby On a 2nd thought, I think this should be in `admin/templates/extending-templates` as a user may not know what terraform is and how the template works.

Reply:
Or maybe it's fine. We can link to a page in `admin/templates/` on how to do this.